### PR TITLE
CSS: update BCD Info

### PR DESCRIPTION
**Pull request by OnkarRuikar**
Same as https://github.com/mdn/content/pull/19185, but for CSS pages.

The PR updates tags, headers, and inline badges as per current BCD v5.1.10.

:notebook: This completes changes in CSS.

***
**Note:** We are simply synchronizing it with BCD. If you have objection with any compatibility status then raise an issue or PR in https://github.com/mdn/browser-compat-data repo.

### CODE REVIEW

File: files/en-us/web/css/list-style-type/index.md [hunk 1]
```diff
@@ -104,75 +104,75 @@ Note that:
...
-- `korean-hanja-formal` {{experimental_inline}}
+- `korean-hanja-formal`
   - : Formal Korean Han numbering.
-- `korean-hanja-informal` {{experimental_inline}}
+- `korean-hanja-informal`
   - : Korean hanja numbering.
 - `lao`, `-moz-lao`
   - : Laotian numbering.
-- `lower-armenian` {{experimental_inline}}\*
+- `lower-armenian`\*
```
Comment (teoli2003) [inline files/en-us/web/css/list-style-type/index.md:143]: What is this `\*` ? It looks like a typo to me.
```suggestion
- `lower-armenian`
```

File: files/en-us/web/css/list-style-type/index.md [hunk 2]
```diff
@@ -104,75 +104,75 @@ Note that:
...
   - : Simplified Chinese informal numbering.
-- `tamil` {{experimental_inline}}, `-moz-tamil`
+- `tamil`, `-moz-tamil`
   - : Tamil numbering.
 - `telugu`, `-moz-telugu`
   - : Telugu numbering.
 - `thai`, `-moz-thai`
   - : Thai numbering.
-- `tibetan` {{experimental_inline}}\*
+- `tibetan`\*
```
Comment (teoli2003) [inline files/en-us/web/css/list-style-type/index.md:165]: Same here
```suggestion
- `tibetan`
```

File: files/en-us/web/css/list-style-type/index.md [hunk 3]
```diff
@@ -104,75 +104,75 @@ Note that:
...
   - : Tibetan numbering.
-- `trad-chinese-formal` {{experimental_inline}}
+- `trad-chinese-formal`
   - : Traditional Chinese formal numbering.
-- `trad-chinese-informal` {{experimental_inline}}
+- `trad-chinese-informal`
   - : Traditional Chinese informal numbering.
-- `upper-armenian` {{experimental_inline}}\*
+- `upper-armenian`\*
```
Comment (teoli2003) [inline files/en-us/web/css/list-style-type/index.md:171]:
```suggestion
- `upper-armenian`
```

**Review by teoli2003 — CHANGES_REQUESTED** [review]

**Review by teoli2003 — APPROVED** [review]
👍

File: files/en-us/web/css/list-style-type/index.md (hunk 1 above)
Comment (OnkarRuikar) [reply files/en-us/web/css/list-style-type/index.md]: I think these were `*` in front of some features indicating something (as in conditions apply stars).
But there was no explanation regarding it, so removing these is viable.

Comment (teoli2003) [reply files/en-us/web/css/list-style-type/index.md]: Yes, I think there was a comment, but it got removed (I have a vague remembrance of having added these 5-8 years ago)

**Pull request merged**